tests/pytests/test_json.py

```diff
@@ -1117,3 +1117,14 @@ def testRedisCommands(env):
         time.sleep(0.1)
         env.expect('JSON.GET', 'doc:1', '$').equal(None)
         env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([0])
+
+no_msan
+def test_mod5608(env):
+    with env.getClusterConnectionIfNeeded() as r:
+        for i in range(1000000):
```

Comment:
Is such a range needed to reproduce?
If so, consider using a pipeline to speed up the test

Reply:
@OfirMos Let's try to avoid unneeded lengthening of the testing duration